Arduino/libraries/Capacitive_Fingerprint_Reader/finger.cpp

```diff
@@ -473,4 +469,10 @@ void Auto_Verify_Finger(void)
 			digitalWrite(Finger_RST_Pin , LOW);    
 		}
 	}
-}
\ No newline at end of file
+}
+
+void serialFlush(){
```

Comment:
Basically if there is anything left in the Serial2 buffer, it places it inside a character that does absolutely nothing. Serial2.read() automatically pushes the string along so doing this until Serial2 is unavailable is the same as emptying out the serial buffer.